.babelrc

```diff
@@ -1,4 +1,4 @@
 {
 	"presets": ["es2015"],
-	"plugins": ["transform-export-extensions"]
+	"plugins": ["transform-export-extensions", "babel-plugin-transform-class-properties"]
```

Comment:
what does this do

Reply:
You can read more on https://medium.com/@jacobworrel/babels-transform-class-properties-plugin-how-it-works-and-what-it-means-for-your-react-apps-6983539ffc22.
In a nutshell, it lets us easily specify arrow functions as member methods in `utils/scroll.js`